lib/rubocop/cop/rails/pluralization_grammar.rb

```diff
@@ -10,25 +10,38 @@ module Rails
       #   # bad
       #   3.day.ago
       #   1.months.ago
+      #   5.megabyte
+      #   1.gigabyte
       #
       #   # good
       #   3.days.ago
       #   1.month.ago
+      #   5.megabytes
+      #   1.gigabyte
       class PluralizationGrammar < Base
         extend AutoCorrector
 
-        SINGULAR_DURATION_METHODS = { second: :seconds,
-                                      minute: :minutes,
-                                      hour: :hours,
-                                      day: :days,
-                                      week: :weeks,
-                                      fortnight: :fortnights,
-                                      month: :months,
-                                      year: :years }.freeze
-
-        RESTRICT_ON_SEND = SINGULAR_DURATION_METHODS.keys + SINGULAR_DURATION_METHODS.values
-
-        PLURAL_DURATION_METHODS = SINGULAR_DURATION_METHODS.invert.freeze
+        SINGULAR_METHODS = { second: :seconds,
+                             minute: :minutes,
+                             hour: :hours,
+                             day: :days,
+                             week: :weeks,
+                             fortnight: :fortnights,
+                             month: :months,
+                             year: :years,
+                             byte: :bytes,
+                             kilobyte: :kilobytes,
+                             megabyte: :megabytes,
+                             gigabyte: :gigabytes,
+                             terabyte: :terabytes,
+                             petabyte: :petabytes,
+                             exabyte: :exabytes,
+                             zettabyte: :zettabytes }
+                           .freeze
```

Comment:
Can you tweak to the following?

```suggestion
        SINGULAR_METHODS = {
          second: :seconds,
          minute: :minutes,
          hour: :hours,
          day: :days,
          week: :weeks,
          fortnight: :fortnights,
          month: :months,
          year: :years,
          byte: :bytes,
          kilobyte: :kilobytes,
          megabyte: :megabytes,
          gigabyte: :gigabytes,
          terabyte: :terabytes,
          petabyte: :petabytes,
          exabyte: :exabytes,
          zettabyte: :zettabytes
        }.freeze
```

Reply:
Looks better, I agree